pages/release-process.md

```diff
@@ -144,6 +144,72 @@ When the vote passes on a release candidate, follow the steps below to complete
 
  8.  Send an email to `dev@fluo.apache.org` announcing new release.
 
+### Test a Fluo release
+
+Below are two common testing strategies to verify a Fluo release candidate:
+
+1. Run the integration tests of projects that depend on Fluo (using the release candidate jars)
+
+1. Run example Fluo applications (using the release candidate tarball)
+
+These strategies are described below with step-by-step instructions.
+
+#### Run integration tests
+
+A Fluo release can be tested by running the integration tests of projects that use Fluo.
+
+1. Before running integration tests, you'll need the following information:
+    * Release version (example: `1.2.0`)
+    * Staging repo (example: `https://repository.apache.org/content/repositories/orgapachefluo-1023/`)
+        * Note: The staging repo changes with each release candidate.
+
+1. Add the following configuration to `~/.m2/settings.xml` to instruct Maven to build using the staging repo
+   for the release candidate. Also, make sure that the `fluoRC` profile is active while you are testing the release
+   but comment out `<activeProfiles>` section after you are done testing.
```

Comment:
This is far too wordy and explanatory. Just put the command to create a temporary settings.xml file, which can be specified on the maven command line later:
```
3. Create a settings.xml file: $ cat <<EOF >/tmp/fluo-its-settings.xml
   ...
EOF
```